### ISSUE TITLE
cherokee currently times out couchdb on large views

### ISSUE THREAD

**Issue by adamgreig** [ukhas/habitat#142]
consider widespread usage of stale=ok or stale=after

**Comment by danielrichman** [comment]
I don't think this is as much of an issue now that we have a cronjob that keeps views relatively up to date; requests are now ready in time. OK to close?
